vault/assets/service_checks.json

```diff
@@ -54,7 +54,7 @@
             "endpoint"
         ],
         "name": "Vault prometheus health",
-        "description": "Returns `CRITICAL` if the check cannot access the metrics endpoint. Returns `OK` otherwise. This service check is only available by using the legacy OpenMetrics implementation (`use_openmetrics: false`)"
+        "description": "Returns `CRITICAL` if the check cannot access the Metrics endpoint. Returns `OK` otherwise. This service check is only available with the legacy version (`use_openmetrics: false)."
```

Comment:
```suggestion
        "description": "Returns `CRITICAL` if the check cannot access the Metrics endpoint. Returns `OK` otherwise. This service check is only available with the legacy version (`use_openmetrics: false`)."
```